src/couchdb/couchdb.service.ts

```diff
@@ -12,48 +15,91 @@ export class CouchdbService {
     private configService: ConfigService,
   ) {}
 
-  get(org: string, path: string, password: string) {
-    const auth = { username: 'admin', password };
-    const url = `https://${org}.${this.domain}/db`;
+  getCouchdb(org: string, password: string) {
+    return new Couchdb(this.http, this.domain, org, password);
+  }
+
+  /**
+   * Run a function for all given databases.
+   * @param credentials
+   * @param callback A function that is called repeatedly, receiving the Couchdb instance for each database;
+   *                  return values are mapped into a key-value return object (key = credentials.name).
+   */
+  async runForAllOrgs(
+    credentials: { name: string; password: string }[],
```

Comment:
Is there a benefit to providing the credentials as parameter? All the calls provide the same credentials anyways.

Reply:
I was thinking to add a query param to controllers in the future in order to let us filter which org it's applied to. (I was struggling to run migration for a single instance for testing.) If that's done in the controller, I think it'll be easier